pkg/services/rendering/rendering.go

```diff
@@ -282,3 +320,20 @@ func isoTimeOffsetToPosixTz(isoOffset string) string {
 	}
 	return isoOffset
 }
+
+func saveMetrics(elapsedTime int64, err error) {
```

Comment:
Might want to distinguish between image and CSV render requests so perhaps should include a label denoting if it's image or CSV?

Reply:
I'm not 100% sure I did it correctly, can you check this commit please: https://github.com/grafana/grafana/pull/33729/commits/3ebbc66104eba570f2bd6502d3298dc09e68a3c9?

Reply:
Looks good :+1: